lua/astrocommunity/completion/supermaven-nvim/init.lua

```diff
@@ -0,0 +1,16 @@
+return {
+  "supermaven-inc/supermaven-nvim",
+  event = "VeryLazy",
+  config = function()
+    require("supermaven-nvim").setup {
+      keymaps = {
+        accept_suggestion = "<C-l>",
+        clear_suggestion = "<C-h>",
+        accept_word = "<C-w>",
+      },
+      log_level = "warn",
+      disable_inline_completion = false, -- disables inline completion for use with cmp
+      disable_keymaps = false, -- disables built in keymaps for more manual control
+    }
```

Comment:
These can be moved to an opts instead.

Reply:
Indeed yes, I just followed the documentation for this but this is better, fixed.